### ISSUE TITLE
command not found

### ISSUE THREAD

**Issue by 12mohaned** [revel/cmd#202]
After installing both packages.

```
go get github.com/revel/cmd/revel
go get github.com/revel/revel
```

When I type revel in terminal, it does output the following message.
**-bash: revel: command not found**

This is my bash_profile. 

export GOPATH=/Users/$USER/go
export PATH="$PATH:$GOPATH/bin"

**Comment by ian-grant** [comment]
I'm a brand-new Revel user, and had the same problem. I solved it by hunting around, and finally finding the "revel" binary (U ran `go build` and `go install`, first)in a weird place due to a typo in my $GOPATH. I then moved the binary to the $GOPATH/bin directory, restarted my shell and it seemed to work!

Because it wasn't in any known path, `which revel` didn't find it, which is why I went hunting for it before realizing my $GOPATH was not pointing in the right direction. Hope this helps!
